### ISSUE TITLE
Return value must be of type mixed, none returned

### ISSUE THREAD

**Issue by thomas-soemoen** [scaramangagency/craftagram#61]
Hei there.

When I did a fresh install in Craft 4 on php 8.1.12 I got the following error when I tried to view the settings:

**/admin/settings/plugins/craftagram**

**scaramangagency\craftagram\Craftagram::getSettingsResponse(): Return value must be of type mixed, none returned**

in line 151
 
```
    public function getSettingsResponse(): mixed {
        Craft::$app->controller->redirect(UrlHelper::cpUrl('craftagram/settings'));
    }
```

 The site is on Craft 4.3.6.1, php 8.1.12 and the plugin is 2.0.1.

**Comment by tomkiss** [comment]
Also seeing this error, unable to install.

**Comment by tomkiss** [comment]
OK, I discovered by uninstalling and reinstalling via the CP that there is a URL issue.

It looks like the two CP links to settings, linked in the logo icon here:

<img width="153" alt="Screenshot 2023-02-03 at 10 31 34" src="https://user-images.githubusercontent.com/183310/216578772-85ebc35c-b824-4739-94db-a4cc0054ae89.png">

And "settings" text here:
<img width="531" alt="Screenshot 2023-02-03 at 10 31 21" src="https://user-images.githubusercontent.com/183310/216578782-579962e3-4246-4b9e-a9d2-92f56406af99.png">

...both link to the wrong place.

The both link to: `/admin/settings/plugins/craftagram`.

If you install via command line, you only ever see links to this broken URL.

Whereas, if you install via CP, after installation you are automatically taken to `/admin/craftagram/settings/1?site`, which appears to work OK.

Hope this is helpful!

**Comment by joshua-martin** [comment]
Fixed in 2.0.2
